components/motor/servo/examples/servo_control/main/servo_control.c

```diff
@@ -0,0 +1,69 @@
+#include <stdio.h>
+#include <math.h>
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "driver/i2c_types.h"
+#include "esp_log.h"
+#include "esp_system.h"
+#include "sdkconfig.h"
+#include "iot_servo.h"
+
+#define SERVO_GPIO          (2)  // Servo GPIO
+
+static const char *TAG = "Servo Control";
+
+uint16_t calibration_value_0 = 30; // Real 0 degree angle
+uint16_t calibration_value_180 = 195; // Real 0 degree angle
+
+
+// Task to test the servo
+static void servo_test_task(void *arg)
+{
+    ESP_LOGI(TAG, "Servo Test Task");
+    while (1) {
+        // Set the angle of the servo
+        for(int i = calibration_value_0; i <= calibration_value_180; i += 1) {
+            iot_servo_write_angle(LEDC_LOW_SPEED_MODE, 0, i);
+            vTaskDelay(20 / portTICK_PERIOD_MS);
+        }
+        // Return to the initial position
+        iot_servo_write_angle(LEDC_LOW_SPEED_MODE, 0, calibration_value_0);
+        vTaskDelay(1000 / portTICK_PERIOD_MS);
+    }
+    vTaskDelete(NULL);
+}
+
+void servo_init(void)
```

Comment:
```suggestion
static void servo_init(void)
```

add static as you only used in this file